devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -387,11 +387,24 @@ private dimmerEvents(physicalgraph.zwave.Command cmd, ep = null) {
 	def value = (cmdValue ? "on" : "off")
 	def result = [createEvent(name: "switch", value: value)]
 	if (cmdValue && cmdValue <= 100) {
+		if(isDINDimmer()){
+			cmdValue = adjustValueToRange(cmd.value)
+		}
 		result << createEvent(name: "level", value: cmdValue == 99 ? 100 : cmdValue)
 	}
 	return result
 }
 
+Integer adjustValueToRange(value){
+	if(value == 0){
+		return 0
+	}
+	def minDimmingLvlPref = getCommandValue(parameterMap.find({it.key == 'minimumDimmingValue'})) ?: parameterMap.find({it.key == 'minimumDimmingValue'}).defaultValue
```

Comment:
if the value has been set it should just be accessible at `settings.minimumDimmingValue`

Reply:
Done